_ext/featured_posts.rb

```diff
@@ -35,7 +35,7 @@ def execute(site)
         # Guarantee that each page has a date value for sorting purposes
         featured.each do |p|
           if p.date.nil?
-            if ( p.relative_source_path =~ /^#{@path_prefix}\/(20[01][0-9])-([01][0-9])-([0123][0-9])-([^.]+)\..*$/ )
+            if ( p.relative_source_path =~ /^#{@path_prefix}\/(20[0123456789][0-9])-([01][0-9])-([0123][0-9])-([^.]+)\..*$/ )
```

Comment:
Would not be easier to change it to `[0-9]`?

Reply:
Good catch; but please just do these kinds of changes yourself, no need for another round of forth and back :) I just changed it and will merge it.